### PR TITLE
More types - Fixes Soil Moisture Sensors

### PR DESCRIPTION
**Pull request by mike240se**
**Describe what the PR does:**
Fixes Soil Moisture Sensors and indoor humidity data type

**Does this fix a specific issue?**
yes, #15 

Fixes https://github.com/bachya/ecowitt2mqtt/issues/15
  
**Checklist:**

- [ ] Update `README.md` with any new documentation.
- [ ] Add yourself to `AUTHORS.md`.

### CODE REVIEW

File: ecowitt2mqtt/const.py [hunk 1]
```diff
@@ -31,16 +31,16 @@
...
+DATA_POINT_SOILHUM1 = "soilmoisture1"
+DATA_POINT_SOILHUM10 = "soilmoisture10"
+DATA_POINT_SOILHUM2 = "soilmoisture2"
+DATA_POINT_SOILHUM3 = "soilmoisture3"
+DATA_POINT_SOILHUM4 = "soilmoisture4"
+DATA_POINT_SOILHUM5 = "soilmoisture5"
+DATA_POINT_SOILHUM6 = "soilmoisture6"
+DATA_POINT_SOILHUM7 = "soilmoisture7"
+DATA_POINT_SOILHUM8 = "soilmoisture8"
+DATA_POINT_SOILHUM9 = "soilmoisture9"
```
Comment (zackbcom) [inline ecowitt2mqtt/const.py:43]:
```suggestion
DATA_POINT_SOILMOISTURE1 = "soilmoisture1"
DATA_POINT_SOILMOISTURE10 = "soilmoisture10"
DATA_POINT_SOILMOISTURE2 = "soilmoisture2"
DATA_POINT_SOILMOISTURE3 = "soilmoisture3"
DATA_POINT_SOILMOISTURE4 = "soilmoisture4"
DATA_POINT_SOILMOISTURE5 = "soilmoisture5"
DATA_POINT_SOILMOISTURE6 = "soilmoisture6"
DATA_POINT_SOILMOISTURE7 = "soilmoisture7"
DATA_POINT_SOILMOISTURE8 = "soilmoisture8"
DATA_POINT_SOILMOISTURE9 = "soilmoisture9"
```

**Review by zackbcom — CHANGES_REQUESTED** [review]
Looks like const.py file didn't get the updated Consts for Soil Moisture.

**Comment by zackbcom** [comment]
image
With my changes on top of all the rest. My moisture sensor is working.

File: ecowitt2mqtt/const.py (hunk 1 above)
Comment (mike240se) [reply ecowitt2mqtt/const.py]: Thanks Zack, I fixed it.

**Pull request merged**